src/sbt-test/scoverage/scala3-coverage-excluded-files/build.sbt

```diff
@@ -4,7 +4,7 @@ scalaVersion := "3.2.0-RC1"
 
 libraryDependencies += "org.scalameta" %% "munit" % "0.7.29" % Test
 
-coverageExcludedFiles := ".*\\/two\\/GoodCoverage"
```

Comment:
How come this changes?

Reply:
These escapes aren't needed on linux, and cause issues on windows.

Forward slashes aren't a special character in regular expressions so they don't need to be escaped. The only reason the test works now is because this string becomes the regex `.*\/two\/GoodCoverage`, and while you don't need to escape forward slashes, an escaped forward slash is just a forward slash and works as expected.

However, on windows our excludedFiles regex needs to look for a backslash instead of a forward slash due to windows using backslashes for paths. However, unlike forward slashes, backslashes *do* need to be escaped in regular expressions. So this commit adds a change to replace `/` with `\\` (i.e. replace a single forward slash with two literal backslashes).

If we do this replacement, with the original regex in the test, the regex on windows becomes `.*\\\two\\\GoodCoverage` with three literal backslashes. This means the regex contains an escaped backslash followed by an escaped `t`, and another escaped backslash followed by an escaped `G`. The escaped `t` and `G` break the regex.

With the changes regex in the test and the new string replace stuff, on linux the regex is `.*/two/GoodCoverage`, and on windows the regex is `.*\\two\\GoodCoverage`.

Reply:
Gotcha thanks for the explanation.